src/models/User.js

```diff
@@ -2,17 +2,14 @@ module.exports = (sequelize, DataTypes) => {
   const User = sequelize.define(
     'User',
     {
-      id: {
-        allowNull: false,
-        autoIncrement: true,
-        primaryKey: true,
-        type: DataTypes.INTEGER,
-      },
       first_name: {
         type: DataTypes.STRING,
         allowNull: false,
 
       },
+      middle_name: {
+        type: DataTypes.STRING
+      },
       last_name: {
```

Comment:
Why did you add `middle_name`, did the team approve this?

Reply:
The middle_name was return my Facebook

Reply:
Is not compulsory that you have to store that in our database if the team didn't include that.

Reply:
OK,  should I remove it. Because I didn't make it a required field.